Doc/prometheus-grafana.md

```diff
@@ -10,6 +10,8 @@ Getting started with Prometheus
 
 This is a configuration example that we will use for this documentation:
 
+**Note**: You will need to add the `m6web/http-kernel-bundle` to listen to the `m6kernel.terminate` and get the timing from the event. 
```

Comment:
will need ? Maybe we can only suggest to install it ?

Reply:
Well, if you want to reproduce this exemple, you will need it, as the `m6kernel.terminate` event comes from it. But I've only added the `m6web/http-kernel-bundle` in the suggest section of the composer file.

Reply:
Can you add a link the the bundle's github home ?